crates/ink/tests/ui/contract/pass/constructor-return-result-err.rs

```diff
@@ -24,18 +24,14 @@ fn main() {
     use contract::Contract;
     use std::any::TypeId;
 
-    const ID: u32 = <Contract as ::ink::reflect::ContractDispatchableConstructors<
-        { <Contract as ::ink::reflect::ContractAmountDispatchables>::CONSTRUCTORS },
-    >>::IDS[0];
```

Comment:
We will need to document the breaking change that you can no longer access constructor/message info in this way by the index. But I doubt too many people are depending on that...

Reply:
Can you please open a separate issue for this?